docs/01_introduction/README.md

```diff
@@ -252,25 +253,25 @@ which is reflected in some of the design choices that were made.
 
 ### *Performance*
 
-We strongly prefer **building software from source**, whenever possible.
+EasyBuild strongly prefers to **build software from source code**, whenever possible.
 
 This is important to ensure that the binaries that are installed can maximally exploit
-the capabilities of processors on which the software will be used.
+the capabilities of the machine architecture on which the software will be run.
 
 For that same reason, EasyBuild **optimizes software for the processor architecture of the build host**
 by default, via compiler options like ``-march=native`` (GCC), ``-xHost`` (Intel compilers), etc.
-This behaviour [can be changed via the ``--optarch`` configuration setting](https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html), if desired.
+This behaviour [may be changed via the ``--optarch`` configuration setting](https://easybuild.readthedocs.io/en/latest/Controlling_compiler_optimization_flags.html).
 
 
 ### *Reproducibility*
 
-In addition to performance, **reproducibility of installations** is another major focus point.
+In addition to performance, **reproducibility of installations** is vital for EasyBuild.
```

Comment:
Hmm, not a big fan of "vital" here...

It's not that EasyBuild requires stuff to be reproducible, it's that we try to make installations reproducible through EasyBuild...

Reply:
Ack. "In addition to performance, **reproducibility of installations** is a core aspect of EasyBuild."

Reply:
OK!